compiler/runtime/OMRCodeCache.hpp

```diff
@@ -313,6 +313,106 @@ class OMR_EXTENSIBLE CodeCache : public CodeCacheBase
     */
    void setFreeBlockList(CodeCacheFreeCacheBlock *fcb) { _freeBlockList = fcb; }
 
+   /**
+    * @brief Getter for the base address of temporary trampolines
+    *
+    * @returns The base address of temporary trampolines
+    */
+   uint8_t *tempTrampolineBase() { return _tempTrampolineBase; }
+
+   /**
+    * @brief Setter for the base address of temporary trampolines
+    *
+    * @param[in] address : The new base address of temporary trampolines
+    */
+   void setTempTrampolineBase(uint8_t *address) { _tempTrampolineBase = address; }
+
+   /**
+    * @brief Getter for the top address of temporary trampolines
+    *
+    * @returns The top address of temporary trampolines
+    */
+   uint8_t *tempTrampolineTop() { return _tempTrampolineTop; }
+
+   /**
+    * @brief Setter for the top address of temporary trampolines
+    *
+    * @param[in] address : The new top address of temporary trampolines
+    */
+   void setTempTrampolineTop(uint8_t *address) { _tempTrampolineTop = address; }
+
+   /**
+    * @brief Getter for the next allocation address of temporary trampolines
+    *
+    * @returns The next allocation address of temporary trampolines
+    */
+   uint8_t *tempTrampolineNext() { return _tempTrampolineNext; }
+
+   /**
+    * @brief Setter for the next allocation address of temporary trampolines
+    *
+    * @param[in] address : The new next allocation address of temporary trampolines
+    */
+   void setTempTrampolineNext(uint8_t *address) { _tempTrampolineNext = address; }
+
+   /**
+    * @brief Getter for the allocation mark for trampolines
+    *
+    * @returns The allocation mark for trampolines
+    */
+   uint8_t *trampolineAllocationMark() { return _trampolineAllocationMark; }
+
+   /**
+    * @brief Setter for the trampoline allocation mark
+    *
+    * @param[in] address : The new trampoline allocation mark
+    */
+   void setTrampolineAllocationMark(uint8_t *address) { _trampolineAllocationMark = address; }
+
+   /**
+    * @brief Adjusts the trampoline allocation mark by the specified amount
+    *
+    * @param[in] numBytes : The number of bytes to adjust the allocation mark by.  Negative
+    *              values are permitted
+    */
+   void adjustTrampolineAllocationMark(size_t numBytes) { _trampolineAllocationMark += numBytes; }
```

Comment:
The documentation says that negative values for `numBytes` are permitted. However, `size_t` is defined as an unsigned type, so it cannot have a negative value. :slightly_smiling_face: 

The fix is probably to just use a different type.

Reply:
Fixed in f7ac529